src/library/returnBook/ReturnBookUI.java

```diff
@@ -1,101 +1,100 @@
 package library.returnBook;
+
 import java.util.Scanner;
 
 
 public class ReturnBookUI {
 
-    private enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
+    private enum UiState {INITIALISED, READY, INSPECTING, COMPLETED};
```

Comment:
#PR42 - D2: Please rename/refactor variable to ReturnItemUIState as per class diagram

Reply:
https://github.com/elliotmitchum/ITC205/pull/42/commits/c565c7508e76f540caef5711aa968f5a9487ae2f